ubiquity/.github
# feat: cli init

Resolves #95 

Comment based parsing as opposed to permit based.

Accounting for the multiple formats going back as far as the start of 2023 I never went back any further thinking it would only be covering the last year.

See readme for usage instructions.

Originally I started with merged pull requests then grabbed the closing issue reference and worked backwards that way but after trying it from an all issues approach it felt like the results were better.

> No we should include every contribution that exists.

So it's no longer just tallying contributions from the last year as per the spec but all time payments?

## QA
- See generated files in root
- Action run: https://github.com/Keyrxng/ubq-airdrop-cli/actions/runs/7860439292
